Assets/MRTK/StandardAssets/packagetemplate.json

```diff
@@ -25,5 +25,8 @@
     },
     "bugs": {
         "url": "https://github.com/microsoft/MixedRealityToolkit-Unity/issues"
+    },
+    "dependencies": {
+        "com.unity.textmeshpro": "2.1.1"
```

Comment:
Similar comment for dependencies of TMP vs what we have here:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Packages/manifest.json#L4

It feels like these numbers should be same, or we should know why they aren't the same.

Reply:
1.4.1 is not provided by the package manager in Unity 2019.4. since we only provide UPM packages when using 2019.4 and newer, it feels prudent to specify a version that is available via the UPM interface. 2.1.1 is the latest verified version of the TMP package for 2019.

Reply:
![image](https://user-images.githubusercontent.com/13281406/95261343-15379e00-07df-11eb-8571-0c5cdf8b5f30.png)